src/main/scala/cromwell/engine/workflow/WorkflowOptions.scala

```diff
@@ -0,0 +1,89 @@
+package cromwell.engine.workflow
+
+import com.typesafe.config.ConfigFactory
+import cromwell.util.{EncryptedBytes, SecretKey, Aes256Cbc}
+import scala.collection.JavaConverters._
+import spray.json._
+import scala.util.{Try, Success, Failure}
+
+object WorkflowOptions {
```

Comment:
Ok. I didn't realize that we'd need to store 2 bits of information for each of these encrypted fields when I first wrote it up. So to see if I understand, if there was an iv column you wouldn't need this json stuff - i.e. is the json just a way to mush the two bits into one string?